mods/cnc/rules/structures.yaml

```diff
@@ -309,6 +336,18 @@ HAND:
 
 AFLD:
 	Inherits: ^BaseBuilding
+	HitShape:
```

Comment:
One or the other, but i'm not sure which is best.  Artwork feels better but is more (possibly too much) work to set up and balance than occupied cells.